doc/classes/Dictionary.xml

```diff
@@ -4,16 +4,63 @@
 		Dictionary type.
 	</brief_description>
 	<description>
-		Dictionary type. Associative container which contains values referenced by unique keys. Dictionaries are always passed by reference.
+		Dictionary type. Associative container which contains values referenced by unique keys. Just like in real life each dictionary composes multiple pairs of unique keys and values. You can define a dictionary by placing a comma separated list of key:value pairs in curly braces [code]{}[/code].
+		[codeblock]
+		var my_dir = {
+    	        key1 : value1,
+    	        key2 : value2,
+    	        key3 : value3
+		}
```

Comment:
There should be a indentation inside this block.

Reply:
done

Reply:
The indentation is too deep, 4 spaces are enough.

Reply:
Are you sure? 

I was under impression that if you do 

```
func my_func():
    # 4 spaces go here 
```

but when you do something like continues value in say array or inside ()

```
var my_array = [
        value1, value2, value3 # 8 spaces go here 
        value4, value5, value6 # 8 spaces go here 
]
```

[As per style here](https://docs.godotengine.org/en/3.1/getting_started/scripting/gdscript/gdscript_styleguide.html#indentation) that says "Use 2 indent levels to distinguish continuation lines from regular code blocks."

Reply:
Things like dictionaries and enums are kinda of a unofficial exception, since they will most of the time be written as a multiline block.

Reply:
That seems very inconsistent to the main docs shouldn't we promote the consistent style? I checked the examples you in docs and you are right dictionaries and enums appear to be exception to the rules. I will start new pr on docs to update this and fix it her eto have only one indetnt